code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -201,13 +198,12 @@
 	amount_per_transfer_from_this = 57
 	list_reagents = list(/datum/reagent/medicine/salbutamol = 10, /datum/reagent/medicine/leporazine = 15, /datum/reagent/medicine/tricordrazine = 15, /datum/reagent/medicine/epinephrine = 10, /datum/reagent/medicine/lavaland_extract = 2, /datum/reagent/medicine/omnizine = 5)
 
-/obj/item/reagent_containers/hypospray/medipen/species_mutator
-	name = "species mutator medipen"
-	desc = "Embark on a whirlwind tour of racial insensitivity by \
-		literally appropriating other races."
-	volume = 1
-	amount_per_transfer_from_this = 1
-	list_reagents = list("unstablemutationtoxin" = 1)
+/obj/item/reagent_containers/hypospray/combat/heresypurge
+	name = "holy water autoinjector"
+	desc = "A modified air-needle autoinjector for use in combat situations. Prefilled with 5 doses of a holy water mixture."
+	volume = 250
+	list_reagents = list(/datum/reagent/water/holywater = 150, /datum/reagent/peaceborg/tire = 50, /datum/reagent/peaceborg/confuse = 50)
+	amount_per_transfer_from_this = 50

```

Comment:
It takes 50u from a beaker using all it's mixture, so it will have a weird float with "/datum/reagent/peaceborg/tire", "/datum/reagent/peaceborg/confuse". Although it might not be subject to change.